blueprints/evolutionapi/template.toml

```
@@ -10,8 +10,9 @@ env = [
   "AUTHENTICATION_API_KEY=${api_key}",
   "AUTHENTICATION_EXPOSE_IN_FETCH_INSTANCES=true",
   "LANGUAGE=en",
-  "CONFIG_SESSION_PHONE_CLIENT=Evolution API",
+  "CONFIG_SESSION_PHONE_CLIENT=Evolution API V2",
```

```suggestion
  "CONFIG_SESSION_PHONE_CLIENT=Evolution API",
```

I don’t think this is necessary and it might cause some confusion. It would also require updating with every new major release.

What was originally there is fine and already makes clear what it refers to. The Evolution API version is also displayed next to the name in the interface.